classes/Http/Controller/DashboardController.php

```diff
@@ -35,20 +35,29 @@ public function showSpeakerProfile()
     /**
      * Check to see if the CfP for this app is still open
      *
-     * @param integer $currentTime
+     * @param integer|\DateTimeInterface $currentTime
      *
      * @return boolean
      */
     public function isCfpOpen($currentTime = null)
     {
-        if (!$currentTime) {
-            $currentTime = strtotime('now');
+        if (! $currentTime) {
+            $currentTime = new \Datetime();
         }
 
-        if ($currentTime < strtotime($this->app->config('application.enddate') . ' 11:59 PM')) {
-            return true;
+        if (! $currentTime instanceof \DateTimeInterface) {
+            $currentTime = new \DateTime('@' . $currentTime);
         }
 
-        return false;
+        $enddate = new \DateTime($this->app->config('application.enddate'));
+        if ($enddate->format('H:i:s') == '00:00:00') {
```

Comment:
Yesterday evening I couldn't find a different solution. Perhaps checking the microseconds might be a way… I'd rather not add an ```endtime```-field in the config. That would make parsing more complicated and scatter the same stuff over two config-entries while it can stay in one…